Gemfile

```diff
@@ -8,6 +8,7 @@ gem 'minitest-rails', '~> 6.0.0'
 gem 'railties', '~> 6.0.0'
 
 group :active_record do
+  gem 'pg'
```

Comment:
Wonder if adding this is what's causing the tests to fail (in the other PR too?)

Reply:
No, it is more related to database_cleaner I guess. On #442, I fixed it by generating unique email for those test cases that uses same email address for users.